browser-test/src/admin/admin_publish.test.ts

```diff
@@ -70,6 +70,57 @@ test.describe(
   },
 )
 
+test.describe(
+  'publishing all programs with disabled programs feature flag on',
+  {tag: ['@uses-fixtures']},
+  () => {
+    const disabledProgram = 'Disabled test program'
+    const publicProgram = 'Public test program'
+    const questionName = 'admin-publish-test-address-q'
+    const questionText = 'admin-publish-test-address-q'
+    // CreateNewVersion implicitly updates the question text to be suffixed with " new version".
+    const draftQuestionText = `${questionText} new version`
+
+    test.beforeEach(async ({page, adminPrograms, adminQuestions}) => {
```

Comment:
question (non-blocking): Are you planning to add more tests here in the future? If not, you can move the logic in the `beforeEach` block into the main test and delete the `beforeEach` block.

Reply:
No, okay deleted `beforeEach`.